.github/workflows/labels.yml

```diff
@@ -0,0 +1,26 @@
+name: Labels
+
+on:
+  pull_request:
+    branches: [master, vara-stage-1, vara-stage-2, vara-stage-3]
+    types: [labeled]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+  build:
+    uses: ./.github/workflows/build.yml
+    # TODO: dispatch this workflow with github action API
+    # for more conditions. for example, if the there is
+    # already a build check exist. (issue-2888)
```

Comment:
> Why there is duplicate build actions?

it is caused by the label `E2`, need to write more logic for avoiding it

Reply:
But this duplicate actions in fact runs twice, isn't it bad?

Reply:
or we can remove `labels.yml` in this PR, then introduce a complete version of it in another PR

Reply:
> or we can remove `labels.yml` in this PR, then introduce a complete version of it in another PR

that would be better